src/query/service/src/servers/http/v1/roles.rs

```diff
@@ -46,14 +47,31 @@ async fn handle(ctx: &HttpQueryContext) -> Result<ListRolesResponse> {
         .default_role()
         .map_or("public".to_string(), |role| role.to_string());
     let mut roles = vec![];
-    for role in user.grants.roles() {
-        let is_current = role == current_role;
-        let is_default = role == default_role;
-        roles.push(RoleInfo {
-            name: role.clone(),
-            is_current,
-            is_default,
-        });
+    if user.is_account_admin() {
```

Comment:
How to test? And why we need this? Please provide summary

Reply:
We are replacing `show roles` in manage mode with `/v1/roles`, and just keeping them consistent.

Reply:
Got cc @flaneur2020 please review this pr